config/settings.yml

```diff
@@ -12,8 +12,8 @@ expected_worker_count: 5 # for okcomputer endpoint
 
 mailer_host: 'localhost'
 
+argo_url: http://localhost:3000
```

Comment:
is this actually used?

Reply:
i see it referenced in the email erb, e.g. https://github.com/sul-dlss/pre-assembly/pull/1284/files#diff-6e5dcc75b9e9652d1202ebedadc436e5e46d2ec6968aa8752d28cf67e82e9af2R12

Reply:
ah, ok.